Assets/Unity3DTiles/B3DMLoader.cs

```diff
@@ -35,6 +35,16 @@ struct FeatureTable
     {
 #pragma warning disable 0649
         public int BATCH_LENGTH;
+        public double[] RTC_CENTER;
+#pragma warning restore 0649
+    }
+
+    struct BatchTable
+    {
+#pragma warning disable 0649
```

Comment:
I believe that the 3D tiles spec permits a more general schema for batch table contents than specifically "Latitute", "Longitude", and "Height".  It appears that any general application specific collection of properties are allowed.  Each may either be a JSON array or a JSON object that refers to a block of data in the corresponding batch table binary body.  Reference: https://github.com/CesiumGS/3d-tiles/tree/master/specification/TileFormats/BatchTable